### PR TITLE
bump to 6.5.9

### PR DESCRIPTION
**Pull request by boegel**
cfr. https://github.com/TACC/Lmod/pull/160

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 still needs to be tested

**Comment by boegel** [comment]
@wpoely86 release is tagged, integration tests pass, good to go

File: Lmod-UGent.spec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 %global macrosdir %(d=%{_rpmconfigdir}/macros.d; [ -d $d ] || d=%{_sysconfdir}/rpm; echo $d)
 
 Name:           Lmod
-Version:        6.5.8
+Version:        6.5.9
```
Comment (wpoely86) [inline Lmod-UGent.spec]: bump to `.10`

**Comment by boegel** [comment]
@wpoely86 bumped to 6.5.10, integration tests still ok

**Pull request merged**